drivers/gpio/gpio_tle9104.c

```diff
@@ -408,8 +408,8 @@ static int tle9104_init(const struct device *dev)
 
 		register_cfg |= TLE9104_CFG_OUT1DD_BIT << i;
 
-		if (!device_is_ready(current->port)) {
-			LOG_ERR("control GPIO %s is not ready", current->port->name);
+		if (!gpio_is_ready_dt(current)) {
+			LOG_ERR("%s: control GPIO is not ready", dev->name);
```

Comment:
Were the changes in what is logged intentional? Before, the log would contain the name of the GPIO controller, which was not ready. Now it contains the name of the tle9104 device, for which a GPIO controller wasn't ready.

Reply:
Yes, this change was intentional. Because gpio_is_ready_dt can be called on a port which is NULL. Which would cause a slight oops in the log statement.